Scripts/Runtime/Configuration/WitSimpleEntityList.cs

```diff
@@ -33,12 +33,19 @@ public WitSimpleEntityList(string entityIdentifier, List<string> words)
 
```

Comment:
we should get the entity name, the keyword, and the synonyms

here we're just doing entity={keyword:synonym, synonym:synonym}

rename the field keywords into synonyms, and add a keyword parameter